### PR TITLE
doc: add meeting minutes for 2017-06-20

### PR DESCRIPTION
**Pull request by gibfahn**
Fixes: https://github.com/nodejs/build/issues/766

### CODE REVIEW

File: doc/templates/build-wg-meeting-2017-06-20.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Next meeting: 11 July 2017
+
+## Present
+* Michael Dawson (@mhdawson)
+* João Reis (@joaocgreis)
+* Gibson Fahnestock (@gibfahn)
+* Michele Capra (@piccoloaiutante)
+* Kunal Pathak (@kunalspathak)
+* Kyle Farnung (@kfarnung)
+* Ali Iyaz Sheikh (@ofrobots)
```
Comment (kunalspathak) [inline doc/templates/build-wg-meeting-2017-06-20.md]: Remove?

File: doc/templates/build-wg-meeting-2017-06-20.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+   * Reaching out to second mac provider (need to follow up)
+   * Building mac images at MacStadium
+   * Ordered parts for ARM 6 build machine backup.
+   * Talked to Tracy/Foundation team about regular tweets (just need to open issue agree on text now)
+   * Talking to Tracy/Foundation about logo/text infra donators can use.
+* João Reis
+  * Added chakracore-nightly support to nodejs-nightly-builder
+* Gibson
+ * Looked at the 8.1.1 issue
+ * Raised issue on nodejs.org re/ sponsorship page
```
Comment (kunalspathak) [inline doc/templates/build-wg-meeting-2017-06-20.md]: need indentation fix-up?

File: doc/templates/build-wg-meeting-2017-06-20.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+### update clang on FreeBSD 10 [#723](https://github.com/nodejs/build/issues/723)
+
+- Michael: Kyle this might be a good issue for you
+- Gibson: should be similar to  https://github.com/nodejs/build/pull/650, you can raise a PR, and if you need a machine to test on (and can’t get a FreeBSD one) you can comment in the Issue or PR.
+
+### VS2015 and VS2017 install issues [nodejs/node#13641](https://github.com/nodejs/node/issues/13641)
+
+- Gibson: maybe we should have a machine in CI with multiple VS versions installed.
+- Joao: I can take a look at getting machines with each, but I don’t think we need one with both. This is a very specific edge case.
```
Comment (mhdawson) [inline doc/templates/build-wg-meeting-2017-06-20.md]: I think Joao was going to take a look to understand the issue and possibly fix the build scripts, not setting up new machines.  He did say that he felt we did not need ones with both.

Comment (joaocgreis) [inline doc/templates/build-wg-meeting-2017-06-20.md]: That is correct. What about:

```
- João: This is a very specific edge case that does not affect end users. I don't think we
need to add this configuration to CI just for this. I will take a better look at the issue.
```

Comment (gibfahn) [inline doc/templates/build-wg-meeting-2017-06-20.md]: Sure, @joaocgreis (or anyone else) if you have corrections just edit the doc and make the changes. This is just a copy of what was in the Google doc (with minor formatting). Change as you like.

**Pull request merged**